### DISCUSSION TITLE
How can I increase readability of tabby menus?

### DISCUSSION THREAD

**Discussion by ovizii** [Eugeny/tabby#8263] [Q&A]
Sorry for the vague description, I couldn't come up with anything better. I'm pretty new to tabby and have already checked its settings but can't figure out how to “fix” the readability of these sections:

![image](https://user-images.githubusercontent.com/837902/233044604-c8bf8118-a42b-4258-8859-ee07bf82abf7.png)

Unless I am hovering above this menu, I wouldn't know there is anything there its basically grey on dark grey.

![image](https://user-images.githubusercontent.com/837902/233044949-d7885763-88be-4d3a-910a-bdc720179ff2.png)

**Comment by vrodionovit** [comment]
I am also interested in this question
